src/main/java/com/arcbees/analytics/client/ClientAnalytics.java

```diff
@@ -60,7 +60,7 @@ public void run() {
     };
 
     @Inject
-    ClientAnalytics(
+    public ClientAnalytics(
```

Comment:
I don't think the library should force the use of Gin. I agree that AnalyticsModule builder is helpful but should not be the only way to create an Analytics instance.